vnext/src/Libraries/Components/View/View.windows.js

```diff
@@ -33,7 +33,7 @@ const View: React.AbstractComponent<
   // [Windows
   invariant(
     // $FlowFixMe Wanting to catch untyped usages
-    props.acceptsKeyboardFocus === undefined,
+    !props || props.acceptsKeyboardFocus === undefined,
```

Comment:
Wondering where we get null/undefined props instead of an empty object

Reply:
It was during testing that I hit this.  And it's likely that UIManager was busted at the time.  So its probably not a normal error condition.